packages/store/addon/-private/system/core-store.ts

```diff
@@ -285,6 +286,29 @@ abstract class CoreStore extends Service {
     }
 
     if (DEBUG) {
+      // support for moduleFor style unit tests
```

Comment:
👍 to this comment, I might tweak it slightly to make it clearer that it's only supporting `moduleFor` with a _restricted resolver_ (because when using `moduleFor('asdf', { integration: true })` it would find our reexports).

Reply:
clarified